common/src/python/inputs/parameter_store.py

```diff
@@ -141,6 +141,30 @@ def get_api_key(self, path_prefix: str) -> str:
 
         return apikey
 
+    def get_redcap_parameters(self, *, base_path: str,
+                              pid: int) -> REDCapParameters:
+        """Pulls URL and Token for the respective REDCap project from SSM
+        parameter store.
+
+        Args:
+            base_path: base path in the parameter store
+            pid: REDCap project ID
+
+        Returns:
+          the REDCap credentials stored at the parameter path
+
+        Raises:
+          ParameterError if any of the parameters are missing
+        """
+
+        if not base_path.endswith('/'):
```

Comment:
get_parameters already adds a `/` if it is not there, so this is redundant

Reply:
Here, I'm adding the REDCap PID to the base path to get the full parameter path for a REDCap project.